flexmeasures/data/models/generic_assets.py

```diff
@@ -137,6 +137,14 @@ def asset_type(self) -> GenericAssetType:
         ),
     )
 
+    def get_path(self, separator: str = ">") -> str:
```

Comment:
Just wondering whether the default should be sandwiched between spaces, for legibility (like in our breadcrumb).

Reply:
I was doubting about it as well. For text, I prefer to shorten strings as much as possible. Also, I was between the following alternatives:
- `>`, as the breadcrumb.
- `/`, as a file path. This could be confusing, perhaps, but It offers a more compact expression.

Reply:
Right now it ends up being used as a column header in a CSV file, right?

Also considerations:

- asset names can quite commonly contain spaces (`NL energy market>Acme Energy>consumption price` vs. `NL energy market > Acme Energy > consumption price`)
- The delimiter could be exposed as a CLI option.

Your call. It all works.

Reply:
Yes, this is used for the column headers in a CSV file.

Given the possibility of having string with spaces in them, I would prefer not to add any extra space.